src/sockperf.cpp

```diff
@@ -3012,13 +3058,20 @@ static int set_sockets_from_feedfile(const char *feedfile_name) {
                 mc_src_ip = std::string(line + m[8].rm_so, line + m[8].rm_eo);
             }
         } else {
-            log_msg("Invalid input in line %s: "
-                    "each line must have the following format: ip:port or type:ip:port or "
-                    "type:ip:port:mc_src_ip. "
-                    "IPv6 addresses must be enclosed in square brackets.",
-                    line);
-            rc = SOCKPERF_ERR_INCORRECT;
-            break;
+            regexpres = regexec(&regexpr_unix, line, NUM_RE_GROUPS, m, 0);
+            if (regexpres == 0) {
+                type = line[0];
+                addr = line + m[1].rm_so - 1;
+                addr.erase(std::remove(addr.begin(), addr.end(), '\n'), addr.end());
```

Comment:
After updating regexp (see my comment for defs.h) this can be simplified
```suggestion
                addr = std::string(line + m[1].rm_so, line + m[1].rm_eo);;
```